src/api/java/mekanism/api/gear/ModuleData.java

```diff
@@ -275,4 +314,61 @@ public ModuleDataBuilder<MODULE> disabledByDefault() {
             return this;
         }
     }
+
+    /**
+     * Enum of flags for module exclusivity channels
+     */
+    public enum ExclusiveFlag {
+
```

Comment:
I don't think the line breaks between the enum values (and also this initial one between the enum declaration and the first javadoc) are needed or all that beneficial.